### DISCUSSION TITLE
Custom colourscheme

### DISCUSSION THREAD

**Discussion by astro-ray** [MM2-0/Kvaesitso#646] [Q&A] [answered]
From what I understand, Kvaesitso can import custom colour schemes as files. Is there any template for this file that I can use to create my colour schemes?

**Comment by MM2-0** [comment, marked as answer]
No, there isn't, because these files aren't intended to be created manually and you should instead use the built-in editor and export the theme from there. Feel free to inspect these files, they are just plain JSON files and you can open them with any text editor, but you will find that they are really confusing, because I have optimized them in a way that allows me to read and write them with as little lines of code as possible, but I didn't pay attention on whether or not people would be able to edit them manually.
